scripts/cibuild.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
We'll probably want to trim the `.sh` extension from this file to align with the other scripts in this repository.